requirements.txt

```diff
@@ -1,4 +1,5 @@
 boto>=2.7
+boto3
```

Comment:
You likely want a minimal version constraint to ensure folks only get boto3 versions after ENA support was added.